src/plugins/intel_gpu/include/intel_gpu/plugin/compiled_model.hpp

```diff
@@ -47,6 +46,9 @@ class CompiledModel : public InferenceEngine::ExecutableNetworkThreadSafeDefault
     Config m_config;
     InferenceEngine::ITaskExecutor::Ptr m_taskExecutor;
     InferenceEngine::ITaskExecutor::Ptr m_waitExecutor;
+
+private:
+    bool is_serializable();
```

Comment:
https://github.com/openvinotoolkit/openvino/pull/13801#discussion_r1017861290
I changed its visibility and name regarding to the coding style. Thank you.